packages/react-draggable-dialog/src/components/DraggableDialogSurface/DraggableDialogSurface.types.ts

```diff
@@ -0,0 +1,10 @@
+import { DialogSurfaceProps } from '@fluentui/react-components';
+
+export type DraggableDialogSurfaceProps = DialogSurfaceProps & {
+  /**
+   * The element that will be draggable.
+   * Only one child is allowed.
+   * @required
+   */
+  children: JSX.Element;
```

Comment:
```suggestion
  children: React.ReactElement;
```

Reply:
This is actually unnecessary, as `DialogSurfaceProps` already has `children`. I removed this explicit type.